src/host/ncp_spinel.hpp

```diff
@@ -404,6 +404,14 @@ class NcpSpinel : public CliDaemon::Dependencies
         mBackboneRouterMulticastListenerCallback = aCallback;
     }
 
+    /**
+     * This method sets the Host Power state on the NCPk.
+     *
+     * @param[in] aState  The Host Power state.
+     */
+
+    otError SetHostPowerState(spinel_power_state_t aState);
```

Comment:
As commented above, we should add an asyncTask here to return the async result when receiving a response from the host.